bin/node/runtime/src/lib.rs

```diff
@@ -468,18 +467,81 @@ impl pallet_sudo::Trait for Runtime {
 	type Call = Call;
 }
 
-/// A runtime transaction submitter.
-pub type SubmitTransaction = TransactionSubmitter<ImOnlineId, Runtime, UncheckedExtrinsic>;
-
 parameter_types! {
 	pub const SessionDuration: BlockNumber = EPOCH_DURATION_IN_SLOTS as _;
 }
 
+impl frame_system::offchain::SigningTypes for Runtime {
+	type Public = <Signature as traits::Verify>::Signer;
+	type Signature = Signature;
+}
+
+impl<LocalCall> frame_system::offchain::CreateSignedTransaction<LocalCall> for Runtime where
+	Call: From<LocalCall>,
```

Comment:
@bkchr shan't we simply name the thing that `decl_outer_dispatch` created `enum OuterCall` or sth to make this a bit easier to grasp? Having conversations about calls is sometimes hard since you always have to specify which one you mean.

Reply:
https://github.com/paritytech/substrate/blob/master/frame/support/src/dispatch.rs#L1658

You can name it whatever you like in the macro.